docs/monitoring/README.md

```diff
@@ -0,0 +1,75 @@
+# Prometheus Monitoring for TF operator
+
+## Install Prometheus in your Kubernetes Cluster
```

Comment:
To be removed. Will add annotation configuration for Prometheus instead, as mentioned here:
https://cloud.google.com/solutions/white-box-app-monitoring-for-gke-with-prometheus